diag_manager/diag_output.F90

```diff
@@ -111,6 +111,8 @@ SUBROUTINE diag_output_init (file_name, file_title, file_unit,&
     integer, allocatable, dimension(:) :: current_pelist
     integer :: mype  !< The pe you are on
     character(len=9) :: mype_string !< a string to store the pe
+    character(len=128) :: tmp_filename      !< Temp variable to store the file_name
```

Comment:
What makes this a temporary variable?  It looks like a file name for a specific type of file.

Reply:
The PR description does a better job of describing this variable than either its name or the description here.